File: python/program/produk_elektronik.py
class ProdukElektronik:
    def __init__(self, id_produk, nama_produk, merk):
        self.__id_produk = id_produk
        self.__nama_produk = nama_produk
        self.__merk = merk

    # Getter
    def get_id_produk(self):
        return self.__id_produk

    def get_nama_produk(self):
        return self.__nama_produk

    def get_merk(self):
        return self.__merk

    # Setter
    def set_id_produk(self, id_produk):
        self.__id_produk = id_produk

    def set_nama_produk(self, nama_produk):
        self.__nama_produk = nama_produk

    def set_merk(self, merk):
        self.__merk = merk

File: php/program/ProdukElektronik.php
<?php

class ProdukElektronik {
    protected $idProduk;
    protected $namaProduk;
    protected $merk;
    protected $gambar; // Atribut baru untuk jalur gambar

    public function __construct($idProduk, $namaProduk, $merk, $gambar) {
        $this->idProduk = $idProduk;
        $this->namaProduk = $namaProduk;
        $this->merk = $merk;
        $this->gambar = $gambar;
    }

    // Getters
    public function getIdProduk() { return $this->idProduk; }
    public function getNamaProduk() { return $this->namaProduk; }
    public function getMerk() { return $this->merk; }
    public function getGambar() { return $this->gambar; } // Getter untuk gambar

    // Setters
    public function setIdProduk($idProduk) { $this->idProduk = $idProduk; }
    public function setNamaProduk($namaProduk) { $this->namaProduk = $namaProduk; }
    public function setMerk($merk) { $this->merk = $merk; }
    public function setGambar($gambar) { $this->gambar = $gambar; } // Setter untuk gambar
}